pkg/services/app/app.go

```diff
@@ -16,11 +16,13 @@ type DeploymentType string
 type SourceType string
 
 const (
-	DeployTypeKustomize DeploymentType = "kustomize"
-	DeployTypeHelm      DeploymentType = "helm"
+	// TODO: use wego.DeployType and wego.SourceType everywhere
```

Comment:
we should use the ones from application_types here instead of defining them again.

Reply:
See the comment on the line below. It would muddy up the intent of this PR. Follow up issue created.